calypso/service.go

```diff
@@ -330,6 +330,17 @@ func (s *Service) ReshareLTS(req *ReshareLTS) (*ReshareLTSReply, error) {
 		return nil, xerrors.Errorf("verifying proof: %v", err)
 	}
 
+	// Iterate through the new roster and update the set of valid peers
+	cl := onet.NewClient(cothority.Suite, ServiceName)
+	var reply updateValidPeersReply
+	for _, srv := range roster.List {
```

Comment:
Shouldn't this be sent to the union of the old and the new roster?

Reply:
Good point. It's not entirely clear to me the message exchanges that happen during the reshare.
Doesn't the protocol execution below involve only the nodes from the new roster? (`roster.GenerateNaryTreeWithRoot()`)

Reply:
Yes, you're right. I was under the impression that you can do a reshare to a new roster that is completely disjoint of the old one. But in fact you need to have a threshold number of old nodes in the new roster, else the secret cannot be reshared. So the new roster includes a threshold of the old roster, but the other nodes from the old roster will not be contacted.

Reply:
Thank you, that makes it clearer. :)